aiida/cmdline/commands/cmd_data/cmd_show.py

```diff
@@ -220,7 +220,7 @@ def _show_xmgrace(exec_name, list_bands):
             'agr', setnumber_offset=current_band_number, color_number=(iband + 1 % MAX_NUM_AGR_COLORS)
         )
         # write a tempfile
-        tempf = tempfile.NamedTemporaryFile('w+b', suffix='.agr')
+        tempf = tempfile.NamedTemporaryFile('w+b', suffix='.agr')  # pylint: disable=consider-using-with
```

Comment:
Idem: Instead of disabling, couldn't we actually not better use a with statement here?

Reply:
as above, what code would you propose?

Reply:
I see now. Didn't notice it was creating a list of handles and then processing them. What the hell :man_facepalming:  OK, fine to leave this as is for now